build-system/tasks/e2e/build-bento-e2e.js

```diff
@@ -0,0 +1,132 @@
+const {compileCss} = require('../css');
+const {
+  buildBinaries,
+  buildExtensions,
+  buildNpmCss,
+} = require('../extension-helpers');
+
+const fastGlob = require('fast-glob');
+const {endBuildStep} = require('../helpers');
+const path = require('path');
+const {existsSync, mkdirSync, writeFile} = require('fs-extra');
+
+/**
+ * Build the Bento end to end page
+ *
+ * @param {!Object} options
+ * @return {!Promise<void>}
+ */
+async function buildBentoE2E(options) {
+  const e2eFixtures = await fastGlob(
+    'extensions/*/1.0/test-e2e/e2e-fixture.js'
+  );
+
+  const isMissingCss =
+    (await fastGlob(`src/bento/components/*/1.0/dist/styles.css`)).length === 0;
+  if (isMissingCss) {
+    await compileCss(options);
+    await buildExtensions({...options, compileOnlyCss: true});
+  }
+
+  const results = e2eFixtures.flatMap((fixturePath) => {
```

Comment:
This flatMap is waaaay too long to be inlined, the `return ... await Promise.all` at the end of the function confused me ("how can you await after returning?" was my first thought) because I didn't realize the return was inside the arrow function. Let's export it to its own named function?